### PR TITLE
Add ukrainian spec and configuration

### PR DESCRIPTION
**Pull request by nemishkor**
Added support for Ukrainian.

### CODE REVIEW

File: rpm/unofficial-jolla-language-pack-uk.spec [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#make sure change these variables to suit your language
+%define CONFLANG uk
+%define LOCNAME uk_UA
+%define RPM_SUFFIX uk
+%define QM_SUFFIX uk
+
+Name: unofficial-jolla-language-pack-%{CONFLANG}
+Version:	0.1.0
```
Comment (martonmiklos) [inline rpm/unofficial-jolla-language-pack-uk.spec:8]: I know it is not documented anywhere, but we used to use the SFOS version here (4.4.0.68 ATM if I am not mistaken), and bump the Release version if a package changed for a single version.

**Comment by martonmiklos** [comment]
@nemishkor  do you want me to do the packaging and publishing to openrepos, or you can handle that?

**Comment by nemishkor** [comment]
@martonmiklos The package is published already https://openrepos.net/content/nemishkor/unofficial-ukrainian-language-pack

**Comment by martonmiklos** [comment]
> @martonmiklos The package is published already https://openrepos.net/content/nemishkor/unofficial-ukrainian-language-pack

Nice job, thanks for the PR!

**Pull request merged**